EIPS/eip-5409.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: 5409
+title: EIP-1155 Non-Fungible Token extension
+description: Allow EIP-1155 to represent Non-Fungible Tokens (tokens who have a unique owner)
+author: Ronan Sandford (@wighawag)
+discussions-to: https://ethereum-magicians.org/t/eip-5409-non-fungible-token-extension-for-eip-1155/10240
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-23
+requires: 165, 1155
```

Comment:
```suggestion
requires: 165, 721, 1155
```

Reply:
hmm, not sure it does
EIP-5409 is only softly attached to EIP-721, it does not attempt to be 100% compatible with it, the only function from EIP-721 it implements is `ownerOf` but it does not even attempt to be 100% compatible on this single function either (See discussion around throwing)

Reply:
I know. It references EIP-721, though, so it belongs in the requires header.